### PR TITLE
Added fixes related to activation

### PR DESCRIPTION
**Pull request by Sergey90K**

### CODE REVIEW

File: app/src/main/java/pl/ludek/smat/home/pool_office_client/presentation/MainActivity.kt [hunk 1]
```diff
@@ -25,105 +23,98 @@ class MainActivity : AppCompatActivity(), View.OnClickListener {
...
+    }
+
+    private fun setDataToLiveData(){
+         dataFromRepository.getDataFromLocalStorage().observe(this, Observer {
+             dataFromSensor.setPoolInfoData(it)
+         })
+     }
+
+    private fun initView(){
+        setContentView(R.layout.activity_main)
```
Comment (tomasz-ludek) [inline app/src/main/java/pl/ludek/smat/home/pool_office_client/presentation/MainActivity.kt:107]: This line has to be removed!
The content view is being set by `setContentView(binding.root)`

File: app/src/main/java/pl/ludek/smat/home/pool_office_client/presentation/MainActivity.kt [hunk 2]
```diff
@@ -25,105 +23,98 @@ class MainActivity : AppCompatActivity(), View.OnClickListener {
...
-         })
-     }
-
-    override fun onClick(v: View?) {
-        when(v?.id){
-            R.id.relayOneOn ->if(relayFirstOnButton.text == relayFirstOnStr){
-                    dataFromRelay.postOneRelayOn().observe(this, Observer {
-                    if(!it.errorRelay){ relayFirstOnButton.text = relayFirstOffStr
+        binding.relayOneOn.setOnClickListener {
+            if( binding.relayOneOn.text == relayFirstOnStr){
```
Comment (tomasz-ludek) [inline app/src/main/java/pl/ludek/smat/home/pool_office_client/presentation/MainActivity.kt:45]: This condition is very poor it require to store multiple redundant variables.
In the next step we will replace all the buttons with ToggleButtons and this problem will disappear,
but next time try to think of something else :)

**Review by tomasz-ludek — CHANGES_REQUESTED** [review]
One thing to fix, one to remember :)

**Comment by Sergey90K** [comment]
The change has been made.

**Review by tomasz-ludek — APPROVED** [review]
Good job!

**Pull request merged**